src/main/scala/org/broadinstitute/dsde/agora/server/webservice/util/GoogleApiUtils.scala

```diff
@@ -17,7 +17,7 @@ object GoogleApiUtils {
   val emailAddress = AgoraConfig.gcsServiceAccountEmail
   val JSON_FACTORY = JacksonFactory.getDefaultInstance
   val httpTransport = GoogleNetHttpTransport.newTrustedTransport
-  val directoryScopes = Collections.singleton(DirectoryScopes.ADMIN_DIRECTORY_GROUP)
+  val directoryScopes = Collections.singleton(DirectoryScopes.ADMIN_DIRECTORY_GROUP_MEMBER_READONLY)
```

Comment:
Did we want to move this into config? Maybe we make a non-1/20 background story for that, as doing it would require a concomitant vault change